AlertaDengue/dados/dbdata.py

```diff
@@ -5,62 +5,72 @@
 """
 from collections import defaultdict
 from datetime import datetime, timedelta
-from typing import Dict, List, Tuple, Callable, Optional
+from typing import Callable, Dict, List, Optional, Tuple
 
-from sqlalchemy import create_engine
-from django.core.cache import cache
-import pandas as pd
-import numpy as np
 import ibis
+import numpy as np
+import pandas as pd
+from django.core.cache import cache
 from ibis import config as cf
+from sqlalchemy import create_engine
+
+from ad_main import settings
 
 # local
 from dados.episem import episem, episem2date
-from ad_main import settings
+from dados.info_states import STATE_NAMES
 
 with cf.config_prefix('sql'):
     cf.set_option('default_limit', None)
 
+
+PSQL_URI = "postgresql://{}:{}@{}:{}/{}".format(
+    settings.PSQL_USER,
+    settings.PSQL_PASSWORD,
+    settings.PSQL_HOST,
+    settings.PSQL_PORT,
+    settings.PSQL_DB,
+)
+
+db_engine = create_engine(PSQL_URI)
+con = ibis.postgres.connect(url=PSQL_URI)
+
+
 # rio de janeiro city geocode
 MRJ_GEOCODE = 3304557
 
 CID10 = {'dengue': 'A90', 'chikungunya': 'A920', 'zika': 'A928'}
 DISEASES_SHORT = ['dengue', 'chik', 'zika']
 DISEASES_NAMES = CID10.keys()
 
-STATE_NAME = {
-    'CE': 'Ceará',
-    'ES': 'Espírito Santo',
-    'MG': 'Minas Gerais',
-    'PR': 'Paraná',
-    'RJ': 'Rio de Janeiro',
-    'SP': 'São Paulo',
-    'RS': 'Rio Grande do Sul',
-    'MA': 'Maranhão',
-    'SC': 'Santa Catarina',
-}
-
-ALL_STATE_NAMES = STATE_NAME.copy()
-# TODO: add missing states here
+STATE_LIST = settings.ACTIVE_STATES
 
-ALERT_COLOR = {1: 'verde', 2: 'amarelo', 3: 'laranja', 4: 'vermelho'}
 
-ALERT_CODE = dict(zip(ALERT_COLOR.values(), ALERT_COLOR.keys()))
+def get_dict_uf(data):
+    dict_uf = {
+        key: value for (key, value) in data.items() if key in STATE_LIST
```

Comment:
I am not a super fan of the usage of global variables inside a function (like `STATE_LIST`)
I would prefer to use something like

```
def get_dict_uf(data, state_list=STATE_LIST):
```

so it means that it will receive a list called state_list, but if it is not given, it will use STATE_LIST as state_list.

but it is just my opinion :)

Reply:
agree with you @xmnlab. It is more elegant. We have created this global variable because it saved us from having to fetch from the database effectively a piece of information that never changed.